### DISCUSSION TITLE
Game Porting Toolkit Version

### DISCUSSION THREAD

**Discussion by IamABonobo** [Whisky-App/Whisky#496] [Q&A] [answered]
I am currently playing GTA V and Metal HUD says the Game Porting Toolkit Version it is using is 1.0 while my terminal says I have installed the 1.04.  When Whisky download the toolkit, does it download the latest version?
How to check which version it is using and how to change it?

**Comment by mavethee** [comment, marked as answer]
That's because Apple only bumped solo their components that are part of the DMG instead whole formula. **Metal HUD reports correct version** and brew will report correct one whenever Apple will update it.

TL;DR You're fine and running the latest.

**Reply by IamABonobo** [reply]
Ok thank you and also, so should I enable DXVK?
Another thing: when playing GTA V, it is using Directx10 and if I set it to 11 and restart the game, it is still running on 10.

**Reply by mavethee** [reply]
@IamABonobo Enabling DXVK uses DXVK instead GPTK.
About DX10 thing, known issue, you can try using DXVK instead as it's not DX12 game.
